server/game/cards/07-WotW/WarriorsOfTheWind.js

```diff
@@ -0,0 +1,31 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+const { CardTypes, Players } = require('../../Constants');
+
+class WarriorsOfTheWind extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Re-arrange participating cavalry characters',
+            condition: () => this.game.isDuringConflict(),
+            gameAction: AbilityDsl.actions.sendHome(context => ({
+                target: context.player.filterCardsInPlay(card => card.hasTrait('cavalry') && card.isParticipating())
+            })),
+            then: {
+                target: {
+                    activePromptTitle: 'Choose characters',
+                    numCards: 0,
+                    //optional: true,
```

Comment:
As soon as this is marked as optional then the "then" part never triggers at all, but it needs to be possible to select 0 characters to move into the conflict.

@jeremylarner It would be great if you could take a look.

Reply:
#3015 seems to fix this